compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/schema/generator/GraphqlSourceModifier.java

```diff
@@ -151,12 +188,11 @@ private AnnotationNode updateAnnotationNode(AnnotationNode annotationNode, Strin
             return annotationNode;
         }
         if (annotationNode.annotValue().isPresent()) {
-            SeparatedNodeList<MappingFieldNode> updatedFields =
-                    getUpdatedFields(annotationNode.annotValue().get(), schemaString);
-            MappingConstructorExpressionNode node =
-                    NodeFactory.createMappingConstructorExpressionNode(
-                            NodeFactory.createToken(SyntaxKind.OPEN_BRACE_TOKEN), updatedFields,
-                            NodeFactory.createToken(SyntaxKind.CLOSE_BRACE_TOKEN));
+            SeparatedNodeList<MappingFieldNode> updatedFields = getUpdatedFields(annotationNode.annotValue().get(),
+                    schemaString);
+            MappingConstructorExpressionNode node = NodeFactory.createMappingConstructorExpressionNode(
+                    NodeFactory.createToken(SyntaxKind.OPEN_BRACE_TOKEN), updatedFields,
+                    NodeFactory.createToken(SyntaxKind.CLOSE_BRACE_TOKEN));
```

Comment:
When sending a PR, let's check whether we have changed lines that are not relevant to the PR, and try not to change them.

Reply:
restored to previous